### ISSUE TITLE
API Level 18 crash onjava.lang.NoSuchFieldError: android.os.Environment.DIRECTORY_DOCUMENTS Firebase test 

### ISSUE THREAD

**Issue by arpu** [MatheusAlvesA/react-native-androide#4]
testing admob integration on Testlab Firebase with Samsung Galaxy S3, API Level 18

i get 
``` Fatal exception

	at com.facebook.react.modules.core.ExceptionsManagerModule.reportFatalException()

FATAL EXCEPTION: mqt_native_modules
com.facebook.react.common.c: Exception in HostObject::get(propName:FileSystem): java.lang.NoSuchFieldError: android.os.Environment.DIRECTORY_DOCUMENTS, stack:
<unknown>@352:390
v@2:1474
<unknown>@347:110
v@2:1474
<unknown>@340:266
v@2:1474
<unknown>@6:58
v@2:1474
d@2:876
global code@364:4
	at com.facebook.react.modules.core.ExceptionsManagerModule.showOrThrowError()
	at com.facebook.react.modules.core.ExceptionsManagerModule.reportFatalException()
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:525)
	at com.facebook.react.bridge.JavaMethodWrapper.invoke()
	at com.facebook.react.bridge.JavaModuleWrapper.invoke()
	at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
	at android.os.Handler.handleCallback(Handler.java:730)
	at android.os.Handler.dispatchMessage(Handler.java:92)
	at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage()
	at android.os.Looper.loop(Looper.java:176)
	at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run()
	at java.lang.Thread.run(Thread.java:841) ```

**Comment by MatheusAlvesA** [comment]
Hey, thanks for this issue report.

This is a problem with the [FileSystem Module](https://github.com/MatheusAlvesA/react-native-androide/wiki/FileSystem) of my library. The DocumentsDir is available only after API level 19, like you can see in the [Docs of Android](https://developer.android.com/reference/android/os/Environment#DIRECTORY_DOCUMENTS).

To work around this i made a hotfix catching the exception and putting null on the DocumentsDir instead.

I just released the version 1.0.2 of my lib with the hotfix. Please update and tell me if it works for you.

**Comment by arpu** [comment]
yeah works thx a lot! awsome work
